lib/requests.js

```diff
@@ -142,3 +104,58 @@ function _getDataAndStatus({result = {}, error = {}}) {
   const statusCode = result.status || error.status;
   return {data, statusCode};
 }
+
+function _deleteAuthorizationHeader({error}) {
+  // delete the Authorization header to prevent
+  // oauth2 headers potentially in logs
+  if(error.response) {
+    const {headers: responseHeaders} = error.response;
+    // Clone the response headers
+    const newHeaders = new global.Headers(responseHeaders);
+    // delete the Authorization header to prevent
+    // oauth2 headers potentially in logs
+    newHeaders.delete('Authorization');
+    const newResponse = new global.Response(error.response, {
+      headers: newHeaders
+    });
+    error.response = newResponse;
+  }
+  return error;
+}
+
+/**
+ * Makes an https request.
+ *
+ * @param {object} options - Options to use.
+ * @param {URL} options.url - A url.
+ * @param {object} [options.json] - JSON for the request.
+ * @param {object} [options.headers] - Headers for the request.
+ * @param {string} options.method - The HTTP method for the request.
+ * @param {object} [options.oauth2] - OAuth2 creds.
+ * @param {object} [options.searchParams] - URL Queries for the request.
+ *
+ * @returns {object} The results from the request.
+ */
+export async function makeRequest({
```

Comment:
How about naming this `makeHttpsRequest`? Just to make it more clear.

Reply:
Addressed here: https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/32/commits/fafbb842b8d9c3280bd85ef1e37cfb03845870a6

also removed 2 deprecated methods from `request.js`